cmd/theatre-envconsul/main.go

```diff
@@ -68,34 +69,31 @@ func main() {
 func mainError(ctx context.Context, command string) (err error) {
 	switch command {
 	case install.FullCommand():
-		files := []string{
-			*installEnvconsulBinary,
-			*installTheatreEnvconsulBinary,
+		files := map[string]string{
+			*installEnvconsulBinary:        "envconsul",
+			*installTheatreEnvconsulBinary: "theatre-envconsul",
 		}
 
 		logger.Log("msg", "copying files into install path", "file_path", *installPath)
-		for _, f := range files {
-			if err := copyExecutable(f, path.Join(*installPath, f)); err != nil {
+		for src, dstName := range files {
+			if err := copyExecutable(src, path.Join(*installPath, dstName)); err != nil {
 				return errors.Wrap(err, "error copying file")
 			}
 		}
 
 	case exec.FullCommand():
-		clusterConfig, err := rest.InClusterConfig()
-		if err != nil {
-			return errors.Wrap(err, "failed to authenticate within kubernetes")
-		}
-
-		logger.Log("event", "vault_login", "host", *execVaultAddress, "backend", *execAuthBackendMountPath, "role", *execAuthBackendRole)
-		vaultToken, err := getVaultToken(clusterConfig.BearerToken, *execVaultAddress, *execAuthBackendMountPath, *execAuthBackendRole)
-		if err != nil {
-			return errors.Wrap(err, "failed to login to vault")
-		}
+		var vaultToken string
+		if *execVaultToken != "" {
```

Comment:
this should be `==` i think. i could only get a token after providing a junk string to `--vault-token` 😆

Reply:
Yep, me being silly. Thanks!